common/src/main/java/com/microsoft/identity/common/internal/util/StringUtil.java

```diff
@@ -67,4 +70,15 @@ public static String convertSetToString(final Set<String> inputSet, final String
 
         return stringBuilder.toString();
     }
+
```

Comment:
Feel confused on the usage between `com.microsoft.identity.common.internal.util.StringUtil` and `com.microsoft.identity.common.adal.internal.util.StringExtensions`.

Reply:
ADAL is for ADAL... longer term we should deprecate the use of the ADAL version.